poetry.lock

```diff
@@ -57,7 +57,7 @@ python-versions = "*"
 
 [[package]]
 name = "certifi"
-version = "2020.12.5"
```

Comment:
If we don't need to update the versions here for branch functionality, let's revert this to master's version.